### PR TITLE
Adapt exhibits for react-map-gl

### PR DESCRIPTION
**Pull request by Apercu**

### CODE REVIEW

File: exhibits/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import 'babel-polyfill';
```
Comment (abmai) [inline exhibits/app.js]: What's the `babel-polyfill` used for here? Maybe a couple of comments to explain its use would help with understanding.

File: exhibits/browserify/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "browserify-exhibit",
+  "version": "0.0.0",
+  "description": "A browserify/react-map-gl exhibit",
+  "scripts": {
+    "link": "(cd .. && rm -f node_modules && ln -s browserify/node_modules)",
```
Comment (ibgreen) [inline exhibits/browserify/package.json]: For what its worth, while this is clever, I think this should be removed. The purpose here is to make it super easy for someone to get started - either with browserify or with webpack. 

The simplest way to do that is to provide two complete example folders. Duplicating a file like app.js is not an issue.

This line is not easy to understand for a lot of people, and will not work/make sense when they try to copy this to start their own app.

File: exhibits/app.js (hunk 1 above)
Comment (ibgreen) [inline exhibits/app.js]: babel-polyfill might not be needed in react-map-gl?

**Comment by abmai** [comment]
TBH aren't `exhibits` and `examples` the same thing? We can just use the example name itself to differentiate and everything can just be `examples`.

File: exhibits/app.js (hunk 1 above)
Comment (Apercu) [inline exhibits/app.js]: ah yeah it might, not rechecked

**Comment by Apercu** [comment]
Yes my bad, will move them

**Pull request merged**